### DISCUSSION TITLE
How to specify CSS properties directly like TailwindCSS arbitrary values.

### DISCUSSION THREAD

**Discussion by rundezv** [master-co/css#23] [Question]
You can specify any property directly in TailwindCSS as follows. Can you do something similar?

```html
<div class="[counter-increment: count]">
  ...
</div>
```

**Comment by 1aron** [comment]
The `counter-increment` you mentioned happens to be a property we haven't built yet, and it will be added later.
As for the API for arbitrary CSS properties, we can extend it immediately, as follows:
```html
<div class="{counter-increment:count}">
    ...
</div>
```
We use `{}` as a group, which happens to give it fallback behavior.
```html
<!-- Master CSS Group Feature -->
<div class="{p:10|20|30|40;text:center;font:red-60/.5}:hover@sm"></div>
```

**Reply by rundezv** [reply]
That's awesome!!! Looking forward to it!

**Reply by 1aron** [reply]
Good news. Completed 🥳 and will be published in two hours.

**Comment by 1aron** [comment]
@takamoso #24 Completed and released. Thx 🥰
